src/bust.py

```diff
@@ -295,10 +353,22 @@ async def ffmpeg_post_hook(e: Exception = None):
         play_lock = asyncio.Lock()
         # Acquire the lock during playback so that on release, play_song() returns
         await play_lock.acquire()
+
+        audio_to_play = None
+        if self._seek_to_seconds is not None:
+            audio_to_play = FFmpegOpusAudio(
+                self.temp_audio_file,
+                options=f"-filter:a volume={config.VOLUME_MULTIPLIER}",
+            )
+            self._seek_to_seconds = None
+        else:
+            audio_to_play = FFmpegPCMAudio(
+                local_filepath,
+                options=f"-filter:a volume={config.VOLUME_MULTIPLIER}",
```

Comment:
Actually, there's something I'm curious about. I noticed we can pass in arbitrary ffmpeg options when playing audio. Instead of cutting the audio with ffmpeg and then reloading the cut audio in separate steps, can't you just pass in the `            "-ss <timestamp>` directly into the options when playing? This would eliminate all the system IO and double ffmpeg pass overhead

Reply:
I see, in that case we can go with this for now and look into faster alternatives later if it becomes a pain point. This feels like something that should be somehow technically possible but might be a pain to figure out in practice.